app/views/courses/_course_table.html.erb

```diff
@@ -0,0 +1,20 @@
+<h2 class="govuk-heading-m" style="border: 3px solid red">
+  <span class="govuk-caption-m">Accredited body</span>
+  Bradford College
```

Comment:
This can be exposed via the backend quiet easily... perhaps a separate PR though.

Reply:
I will do this on a follow-up PR - keep this one as a refactoring.